### PR TITLE
[auto_submit] Migrate to appengine

### PR DESCRIPTION
**Pull request by CaseyHillers**

### CODE REVIEW

File: auto_submit/lib/server/exceptions.dart [hunk 1]
```diff
@@ -0,0 +1,69 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
```
Comment (keyonghan) [inline auto_submit/lib/server/exceptions.dart:1]: Nit: 2022.

File: cloud_build/deploy.sh [hunk 2]
```diff
@@ -7,10 +7,18 @@
...
 gcloud app deploy --project "$1" cron.yaml
 gcloud app services set-traffic default --splits version-$2=1 --quiet
 # Google Cloud quota only allows 30 versions.
 # For daily builds, this will keep the 10 most recent versions, but wipe the rest.
 gcloud app versions list --format="value(version.id)" --sort-by="~version.createTime" | tail -n +11 | xargs -r gcloud app versions delete --quiet
 popd > /dev/null
+
+# auto_submit
+pushd auto_submit > /dev/null
+gcloud app deploy --project "$1"--version "version-$2" -q "$3" --no-stop-previous-version
```
Comment (keyonghan) [inline cloud_build/deploy.sh:21]: `"$1"--version` -> `"$1" --version`

Comment (keyonghan) [inline cloud_build/deploy.sh:21]: How do we distinct the servers between `auto_submit` and `app_dart`?

File: auto_submit/lib/server/exceptions.dart (hunk 1 above)
Comment (CaseyHillers) [reply auto_submit/lib/server/exceptions.dart:1]: Per https://github.com/flutter/cocoon/pull/701#discussion_r395447338 - this should remain 2019

File: cloud_build/deploy.sh (hunk 2 above)
Comment (CaseyHillers) [reply cloud_build/deploy.sh:21]: https://auto-submit-dot-flutter-dashboard.appspot.com

Comment (CaseyHillers) [reply cloud_build/deploy.sh]: Nice catch, done

**Comment by KristinBi** [comment]
Thanks Casey, I will add these codes to my new pr later once it is merged!

**Pull request merged**